src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Documents/SelectionWordBreaker.cs

```diff
@@ -226,9 +228,7 @@ private static CharClass[] GetClasses(char[] text)
                 }
                 else
                 {
-                    UInt16[] charType1 = new UInt16[1];
-
-                    SafeNativeMethods.GetStringTypeEx(0 /* ignored */, SafeNativeMethods.CT_CTYPE1, new char[] { ch }, 1, charType1);
+                    SafeNativeMethods.GetStringTypeEx(0 /* ignored */, SafeNativeMethods.CT_CTYPE1, new(in ch), charType1);
```

Comment:
Can you specify the type here and in the other places in your changes:
```suggestion
                    SafeNativeMethods.GetStringTypeEx(0 /* ignored */, SafeNativeMethods.CT_CTYPE1, new ReadOnlySpan<char>(in ch), charType1);
```